docs/section22/Rule22-9.md

```diff
@@ -0,0 +1,65 @@
+
+# CHW&CW - Rule 22-9  
+
+**Schema Version:** 0.0.10  
+**Mandatory Rule:** True  
+**Rule ID:** 22-9  
+**Rule Description:** For Baseline chilled water system with cooling capacity of 300 tons or more, the secondary loop shall be modeled with a minimum flow of 25% of the design flow rate.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** Section G3.1.3.10 Chilled-water pumps (System 7, 8, 11, 12 and 13)  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, 13, 7b, 8b, 11b, 12b, 13b.
+2. B-RMR is modeled with primary-secondary configuration.
+3. B-RMR chilled water system cooling capacity is 300 tons or more.
+
+**Function Calls:**  
+
+1. get_baseline_system_types()
+2. get_primary_secondary_loops()
+3. get_component_by_id()
+
+**Applicability Checks:**  
+
+- **Check 1:** Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+
+  - Check if B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b, i.e. with air-side system served by chiller(s), continue to the next applicability check: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-12", "SYS-13", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-12B", "SYS-13B"]): NEXT_APPLICABILITY_CHECK`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+- **Check 2:** Get primary and secondary loops for B-RMR: `primary_secondary_loop_dictionary = get_primary_secondary_loops(B_RMR, "COOLING")`
+
+  - Check if B-RMR is modeled with primary-secondary configuration, continue to rule logic: `if LEN(primary_secondary_loop_dictionary) != 0: CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+## Rule Logic:  
+
+- For each pump in B_RMR: `for pump_b in B_RMR.RulesetModelInstance.pumps:`
+
+  - Save loop served by pump and pump to a dictionary: `loop_pump_dictionary[pump_b.loop_or_piping].append(pump)`
+
+- For each chiller in B_RMR: `for chiller_b in B_RMR.RulesetModelInstance.chillers:`
+
+  - Get cooling loop that chiller serves: `chw_loop_b_id = chiller_b.cooling_loop`
+
+    - Add chiller rated capacity to primary CHW loop total capacity: `chw_loop_capacity_dict[chw_loop_b_id] += chiller_b.rated_capacity`
+
+- For each primary CHW loop: `for loop_b_id in primary_secondary_loop_dictionary["PRIMARY"]:`
+
+  - Check if CHW primary loop total cooling capacity is 300 tons or more: `if chw_loop_capacity_dict[loop_b_id] >= 300 * CONVERSION(TON_TO_BTUH):`
+
```

Comment:
@xingcx and @weilixu the chiller rated capacity is in W in the RMR. Should the conversion here say  conversion (W_to_TON) and the check be >300 Tons?

Reply:
The 300ton is from 90.1 table, and it is converted to Btuh as  we assumes the chiller rated capacity in IP unit is Btuh.

Reply:
I don't think we should assume the IP units for RDS. The RDS is reading the RMR which is in SI units. The RDS should identify the required conversion from the units listed in the schema to the units required for the RDS verification- i.e. the units in Standard 90.1

Reply:
@supriyagoel I believe we agreed that when RMR files are parsed by RCT, all metric values will be converted to IP. However for capacity, this default conversion would be from W to Btu/hr, so @xingcx included conversion from 300 ton to Btu/hr for apples to apples comparison.